app/models/related_content.js

```diff
@@ -1,17 +1,90 @@
 var fs = require('fs');
+var process = require('process');
+var https = require('https');
+var querystring = require('querystring');
 var Promise = require('bluebird');
+var TaxonomyData = require('./taxonomy_data.js');
 var readFile = Promise.promisify(fs.readFile);
+var Taxon = require('./taxon.js');
 
 class RelatedContent {
-  static get (contentBasePath) {
-    var readSourceData = readFile('app/data/hardcoded_related_content.json');
+  static esRelatedLinks (contentBasePath, parentTaxon) {
+    var queryParams = {
+      similar_to: contentBasePath,
+      start: 0,
+      count: 5,
+      filter_taxons: [parentTaxon],
+      fields: 'title,description,link'
+    }
+    var queryString = querystring.stringify(queryParams);
+    var path = "/api/search.json?" + queryString;
+
+    return new Promise((resolve, reject) =>
+      https.get({
+        host: 'www-origin.integration.publishing.service.gov.uk',
```

Comment:
The corresponding rummager branch isn't deployed to production unfortunately: https://github.com/alphagov/rummager/compare/more-like-this-queries?expand=1

If I have time, I will fix the branch and deploy otherwise we will have to do it perhaps in staging.